pyquil/api.py

```diff
@@ -315,7 +260,6 @@ def version(self):
         :rtype: string
         """
         raise DeprecationWarning("Version checks have been deprecated.")
-        return None
```

Comment:
Why remove the DeprecationWarning here?

Reply:
I'm actually just removing the `return None` which was an unreachable statement